### PR TITLE
Fix/4224  integration fixes for advanced search

### PR DESCRIPTION
**Pull request by asf-algoo**
#4224 

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified
- [x] Original authors of the features included in a multi-feature branch (maintenance fixes -> develop, security fixes -> develop, …) should be part of the reviewers, especially if you encountered merge conflicts.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

To test:
- Verify that a file name has its extention at:
    - Recent Activities (dashboard)
    - Search (advanced)
    - Search (simple)
    - Content list ("all contents")
    - App file
- Verify if result's breadcrumbs at an advanced search:
    - doesn't disappear when opening the filters
    - works at all screen types (overflow)
    - has the space name
- Last modification type is translated at advanced search and activity feed
- The information column in advanced search has aligned elements (for all types)

### CODE REVIEW

**Review by raphj — APPROVED** [review]

File: functionnal_tests/cypress/integration/app_file/file_name_spec.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+const threadTitle = 'Thread'
+const fileTitle = 'File'
+const fullFilename = 'Linux-Free-PNG.png'
+const contentType = 'image/png'
+let threadId
+let fileId
+let workspaceId
+
+describe("An app's name", () => {
```
Comment (Skylsmoi) [inline functionnal_tests/cypress/integration/app_file/file_name_spec.js:11]: I don't understand this label, shouldn't it be "a content's name" ?

File: functionnal_tests/cypress/integration/app_file/file_name_spec.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+      pageName: PAGES.CONTENT_OPEN,
+      params: { workspaceId: workspaceId, contentType: 'file', contentId: fileId }
+    })
+    cy.get('.newVersionBtn').click()
+    cy.dropFixtureInDropZone(fullFilename, contentType, '.filecontent__form', fullFilename)
+    cy.get('.file__contentpage__dropzone__btn__validate').click()
+    cy.contains('.FilenameWithExtension .badge', '.png')
+  })
+
+  it('should not show the file extension with content type is not a file', () => {
```
Comment (Skylsmoi) [inline functionnal_tests/cypress/integration/app_file/file_name_spec.js:42]: I'd like to point out that this test doesn't seem related to the app file since it opens the app thread.
You are actually testing the component FilenameWithExtension in this file if I'm right.
Since this file is in frontend_lib, I don't know where to put it though. So, no changes required

File: functionnal_tests/cypress/integration/app_file/file_name_spec.js (hunk 1 above)
Comment (raphj) [reply functionnal_tests/cypress/integration/app_file/file_name_spec.js:11]: Maybe "The app File title"?

File: functionnal_tests/cypress/integration/app_file/file_name_spec.js (hunk 2 above)
Comment (gbonaspetti) [reply functionnal_tests/cypress/integration/app_file/file_name_spec.js:42]: Maybe put at `app_interface`?

File: functionnal_tests/cypress/integration/app_file/file_name_spec.js (hunk 1 above)
Comment (gbonaspetti) [reply functionnal_tests/cypress/integration/app_file/file_name_spec.js:11]: Yes I was thinking "a content's name inside an app", I'll make it clearer.

File: functionnal_tests/cypress/integration/app_file/file_name_spec.js (hunk 2 above)
Comment (raphj) [reply functionnal_tests/cypress/integration/app_file/file_name_spec.js:42]: that would go to a new folder maybe called "apps" to me if we want to handle this, but I'm fine with keeping it in app_file or putting it in app_interface

**Review by grignards — APPROVED** [review]

File: functionnal_tests/cypress/integration/app_file/file_name_spec.js (hunk 2 above)
Comment (grignards) [reply functionnal_tests/cypress/integration/app_file/file_name_spec.js:42]: There is a `generic_frontend` folder in integration.

Comment (Skylsmoi) [reply functionnal_tests/cypress/integration/app_file/file_name_spec.js:42]: indeed, but I think this folder is related to components for the frontend/ folder, not the frontend_lib/ one.

I'm fine with app_interface too

**Review by Skylsmoi — APPROVED** [review]

**Comment by raphj** [comment]
tested on branch
